app/src/services/conditional_token.ts

```diff
@@ -33,11 +32,14 @@ class ConditionalTokenService {
     } else {
       this.contract = new ethers.Contract(address, conditionalTokensAbi, provider)
     }
-    this.address = address
     this.signerAddress = signerAddress
     this.provider = provider
   }
 
+  getAddress = (): string => {
```

Comment:
It wasn't necessary to change the _usages_ of this. If you use a getter like in the comment (repeated here):

```typescript
get address(): string => { ... }
```

then the usage continues to be `marketMaker.address` (without parentheses).

That being said, if you prefer explicit getters, that's OK with me.

Reply:
Mmmm ok , I apply getter syntax.